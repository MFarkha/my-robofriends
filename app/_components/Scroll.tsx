import React from "react";
import { ReactNode } from "react";

type ScrollProps = {
  children?: ReactNode;
};

const Scroll = (props: ScrollProps) => {
  return (
    <div
      style={{
        overflowY: "scroll",
        border: "5px solid black",
        height: "800px",
      }}
    >
      {props.children}
    </div>
  );
};

export default Scroll;
