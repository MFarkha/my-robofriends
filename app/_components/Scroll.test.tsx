import React from "react";
import Scroll from "./Scroll";
import { expect, test } from "vitest";
import { render } from "@testing-library/react";

test("expects to render the Scroll component", () => {
  const { container } = render(<Scroll />);
  expect(container).toMatchSnapshot();
});
