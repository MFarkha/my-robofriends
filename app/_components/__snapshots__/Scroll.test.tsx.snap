// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`expects to render the Scroll component 1`] = `
<div>
  <div
    style="overflow-y: scroll; border: 5px solid black; height: 800px;"
  />
</div>
`;
